.travis.yml

```diff
@@ -1 +1,14 @@
-language: java
+language: android
+jdk: oraclejdk8
+
+android:
+  components:
+    - android-23
+    - build-tools-23.0.1
+    - extra-android-m2repository
+    - extra-android-support
```

Comment:
There is an Android library module `example`, I used for showcase the usage of `auto-value-gson`,  if I remove these lines, travis check will fail. If you think the `example` module is unnecessary, I can remove it at all :)